cmd/rpcdaemon/cli/config.go

```diff
@@ -342,71 +342,66 @@ func RemoteServices(ctx context.Context, cfg httpcfg.HttpCfg, logger log.Logger,
 		}
 
 		// Configure sapshots
-		if cfg.Snap.Enabled {
```

Comment:
Was this condition removed by mistake?

Reply:
Because we completely removes `NewBlockReader`, that's why I removed the check as I removed the else of it:
https://github.com/ledgerwatch/erigon/pull/6754/files#diff-817ffff9639d28a33b7a5b0645eb3493ac7f258c6c3cc5c4cebb6e7ee8865749L406